src/store/modules/user.js

```diff
@@ -0,0 +1,23 @@
+import userState from '../../utils/auth/state/userState';
```

Comment:
This file is duplicating functionality @nicoburniske made /utils/auth/state/userState